src/extension-runners/chromium.js

```diff
@@ -256,6 +256,23 @@ export class ChromiumExtensionRunner {
       chromeFlags.push(...this.params.args);
     }
 
+    // `--remote-debugging-pipe` results in `navigator.webdriver == true` in
+    // the launched Chromium instance (unless a Chromium session exists
+    // already, in which case the existing session is used, which typically
+    // has `navigator.webdriver == undefined`). This breaks websites with bot
+    // detection features. The following flag prevents this.
+    // Only pass this flag if `--enable-blink-features=AutomationControlled`
+    // is not passed.
+    if (
+      !chromeFlags.some(
+        (flag) =>
+          flag.startsWith('--enable-blink-features') &&
+          flag.match(/AutomationControlled/),
```

Comment:
Generally the condition on these lines is broader than needed, but given the length and content of the strings, it is unlikely to match something unrelated, so I am okay with that.

If we wanted to make it very precise, we could change the whole check to `/^--enable-blink-features=(.*,)AutomationControlled(,|)/.test(flag)`.

If we want to avoid unnecessary use of rege, this line here can change to `flag.includes("AutomationControlled"),`.